common/changes/@uifabric/styling/edwl-2018-10-fixCoachmarkDelay_2018-10-03-21-59.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/styling",
+      "comment": "add delayLength parameter to _createDefaultAnimation",
```

Comment:
Nit: a little unclear where this `_createDefaultAnimation` in the styling package is located.